features/flows/classification/engine/impl/src/main/java/org/opennms/netmgt/flows/classification/internal/decision/Tree.java

```diff
@@ -452,11 +454,11 @@ public String toString() {
         }
 
         public final static class WithClassifiers extends Leaf {
-            public final List<Classifier> classifiers;
+            public final Classifiers classifiers;
 
             public WithClassifiers(List<Classifier> classifiers) {
                 super(Info.forLeaf(classifiers.size()));
-                this.classifiers = classifiers;
+                this.classifiers = new Classifiers(true, classifiers);
```

Comment:
What does this do? Why is this change required?

Reply:
The change reduces allocations during classification.